.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Ensure your title is short, descriptive, and in the imperative mood (Fix X, Change Y, instead of Fixed X, Changed Y).
```

Comment:
Let's add a link to this post here:
```suggestion
Ensure your title is short, descriptive, and in the imperative mood (Fix X, Change Y, instead of Fixed X, Changed Y).
For a good inspiration of what to write in commit messages and PR have a look at https://chris.beams.io/posts/git-commit/
```